addons/web/controllers/main.py

```diff
@@ -708,7 +708,7 @@ def backup(self, master_pwd, name, backup_format = 'zip'):
     def restore(self, master_pwd, backup_file, name, copy=False):
         try:
             data = base64.b64encode(backup_file.read())
-            request.session.proxy("db").restore(master_pwd, name, data, copy)
+            request.session.proxy("db").restore(master_pwd, name, data, bool(copy))
```

Comment:
Yes, it's better to parse it correctly.

internal should keep a copy of the method because it should be compatible with 8.0 branch